src/project.js

```diff
@@ -51,4 +50,5 @@ const getItems = product => {
   return cachedProjects;
 };
 
+exports.buildItem = buildItem;
```

Comment:
If I'm not wrong this function is not used outside of this file, actually there is no needs to export it

Reply:
same with the conversation above, reverting the changes